test/quantization/core/test_quantized_op.py

```diff
@@ -4931,6 +4945,85 @@ def test_qconv2d_add(
                 Y_scale, Y_zero_point, use_bias, "add", use_channelwise, False,
                 input_dtype=X_qdtype, output_dtype=X_qdtype, X2_scale=X2_scale, X2_zero_point=X2_zero_point)
 
+    @given(batch_size=st.integers(1, 3),
```

Comment:
please don't add new calls to hypothesis, it has caused a lot of flaky test errors in CI before, can you change them to loops instead?

Reply:
Thanks for the comments. I have changed it with `itertools.product` to use for loop. Please help to take a look again @jerryzh168.